public/app/core/services/backend_srv.ts

```diff
@@ -447,6 +447,7 @@ export class BackendSrv implements BackendService {
           url,
           type,
           redirected,
+          config: options,
           request: { url, ...init },
```

Comment:
I just remembered another thing. Should we keep both request and config?

Reply:
yea, good point, apparently request did not exist before, it as just copied from config in the query inspector: https://github.com/grafana/grafana/blob/v6.6.2/public/app/features/dashboard/panel_editor/QueryInspector.tsx#L100

Reply:
yes, I added this after we talked about it back in the day :)

Reply:
request is doing something a bit different, parsing the options body for example (rending as json string) instead of the original data object, and not including everything, it's going through https://github.com/grafana/grafana/blob/8e1fe6a9dd7a1f95365dcfdee50f2b1d43754bb0/public/app/core/utils/fetch.ts#L4  
https://github.com/grafana/grafana/blob/master/public/app/core/services/backend_srv.ts#L450

in 6.7
![image](https://user-images.githubusercontent.com/10999/77513137-b75fec00-6e74-11ea-9b80-ea1b98a1257c.png)

in 6.6:
![image](https://user-images.githubusercontent.com/10999/77513260-f4c47980-6e74-11ea-922a-1c819e7f4d25.png)

Will try to change it so that we only return the unmodified config object. We will miss a bit of info (so not see how data/params are serialized) but think it's better to see just what was passed to backendSrv more clearly. What do you think?

Reply:
I agree with that!